inference/worker/interface.py

```diff
@@ -13,23 +13,19 @@ class GenerateStreamParameters(pydantic.BaseModel):
     temperature: float | None
     repetition_penalty: float | None
     seed: int | None
-    stop: list[str] = [
-        "<|endoftext|>",
-        "<|prompter|>",
-        "<|assistant|>",
-    ]  # TODO: make this a bit more workable because it's mutliple tokens
+    stop: list[str] = []
```

Comment:
normal stop should always the eos token of the tokenizer ..

Reply:
that is happening anyway, independent of the tokens here. Moreover, putting eos token here confuses the stopping criteria, making it e.g. impossible to generate html